EIPS/eip-5XXX.md

```diff
@@ -0,0 +1,341 @@
+
+The optional `mint` method was also included to similarly cover the creation of Principal Tokens. Though a ubiquitous pattern for minting is less present in current implementations, and is less necessary than `redemption` for common use cases, the ability to directly mint Principal Tokens reduces the complexity of integrations.
+
+The optional `maturityRate` and `exchangeRate` methods are included to allow for the accommodation of additional yield generation post-maturity. In implementations that include these optional methods, the amount of underlying returned by the `redeem` method is equivalent to ```amount + (amount * (exchangeRate/maturityRate))```.
+
+## Backwards Compatibility
+
+ERC-5070 is fully backward compatible with the ERC-20 standard and has no known compatibility issues with other standards.
+For production implementations of Principal Tokens which do not use ERC-5070, wrapper adapters can be developed and used, or wrapped tokens can be implemented.
+
+## Reference Implementations
+
+Official Example Implementation: https://github.com/JTraversa/Principal-Token-Standard
+
+Yield fyTokens: https://github.com/yieldprotocol/vault-v2/blob/master/contracts/FYToken.sol
+
+Swivel zcTokens: https://github.com/Swivel-Finance/swivel/blob/main/contracts/v2/marketplace/ZcToken.sol
+
+Element Principal Tokens: https://github.com/element-fi/elf-contracts/blob/main/contracts/Tranche.sol (maturity->unlockTimestamp)
+
+## Security Considerations
+
+Security considerations surrounding fees?
+
+Fully permissionless use cases could fall prey to malicious implementations which only conform to an ERC-5070 interface but not the specification, failing to implement proper custodial functionality but offering the ability to purchase Principal Tokens through secondary markets.
+
+It is recommended that all integrators review each implementation for potential ways of losing user deposits before integrating.
+
+The methods `totalAssets`, `convertToShares` and `convertToAssets` are estimates useful for display purposes,
+and do _not_ have to confer the _exact_ amount of underlying assets their context suggests. _Alberto: This might be a remnant of ERC4626, which might be useful to explore. Maybe we use the same pattern instead of `exchangeRate`._
+
+As is common across many standards, it is strongly recommended to mirror the underlying token's `decimals` if at all possible, to eliminate possible sources of confusion and simplify integration across front-ends and for other off-chain users.
+
+## Copyright
+
+Copyright and related rights waived via [CC0](https://creativecommons.org/publicdomain/zero/1.0/).
```

Comment:
We have a new copyright line.  Please copy from the latest version of this line from https://raw.githubusercontent.com/ethereum/EIPs/master/eip-template.md

Reply:
Fixed, thank you!